src/rust/server/segcache/src/main.rs

```diff
@@ -55,6 +55,11 @@ fn main() {
                 .help("Server configuration file")
                 .index(1),
         )
+        .arg(
+            Arg::with_name("print-config")
+                .help("list & describe all options in config")
```

Comment:
let's make the help text more consistent with the rest of the help text in the rust implementation. I would prefer "List all options in config", also the option should be `--config` with `-c` as a short form to be consistent with the C version.